### ISSUE TITLE
The issue of rendering quality. 

### ISSUE THREAD

**Issue by ingra14m** [hustvl/4DGaussians#3]
Thanks to the outstanding work by the authors. 

During my experiments, I attempted various approaches for modeling the spatial changes in 3D-GS. Based on my experimental results, grid/plane-based methods appear to be more suitable for directly inputting a feature (128 or 256 dim) into an MLP as the encoding of time. Using such structures to learning delta for (position, rotation and scaling) seems to have a negative impact since the introduction of more learnable parameters. 

I'm not sure if the authors have optimized the Grid/Plane structure for GS, and I look forward to the your response.

**Comment by guanjunwu** [comment]
Thank you for the interesting discussion on adopting the neural voxel grids&planes !

I've also tried various methods to encode the time features, such as the pure implicit of your proposed paper, MLP+Voxel Grids like TiNeuVox, or multi-resolution hash planes implemented in Tinycudann, but most of the methods can work very well. 

As is shown in Fig.4 of our paper, though total learnable parameters may increase, the relationship between each 3D Gaussian and deformation field only depends on nearby multi-resolution voxels and MLP. It seems like the learnable parameters of each 3D Gaussian become smaller and faster to converge. And you can check the implementation in the code.

Meanwhile, a proper hyperparameter setting may contribute to joint optimization. When I increase the learning rate of grid > 0.1 and MLP > 0.01, the total deformation field may fail.

Now I'm trying to design a better grid/plane structure for the Gaussians deformation field, welcome to discuss the details of deformation fields together :).
